questionary/prompts/common.py

```diff
@@ -188,6 +188,7 @@ def __init__(
         self,
         choices: Sequence[Union[str, Choice, Dict[str, Any]]],
         default: Optional[Union[str, Choice, Dict[str, Any]]] = None,
+        pointer: str = DEFAULT_SELECTED_POINTER,
```

Comment:
Can you add a description in the docstring for this argument?

I think perhaps we could remove the `use_pointer` option and just set `pointer` to `None` to indicate that the pointer should be hidden.

Reply:
Sure it would be less redundant